### DISCUSSION TITLE
can't find v-data-table v3

### DISCUSSION THREAD

**Discussion by adverich** [vuetifyjs/vuetify#16421] [Q&A] [answered]
Why isn't v-data-table available in v3? I don't even see it in the docs....

**Comment by santiagodeheza** [comment, marked as answer]
Its part of `v3.1` and labs. (new way for developers to use unfinished components in an alpha state)
[https://next.vuetifyjs.com/en/labs/introduction/#usage](https://next.vuetifyjs.com/en/labs/introduction/#usage)

**Reply by WayneDT** [reply]
How do I install the labs?
In the doco it states "import { VDataTable } from 'vuetify/labs/VDataTable'" but the labs folder does not exist.
![image](https://user-images.githubusercontent.com/88186111/216846915-8f0d19c1-4f7b-42b1-9785-8154fcb59897.png)

I do see VDataTable under "lib/components" but am unable to import it as it gives errors.
thanks

**Reply by suruaku** [reply]
https://next.vuetifyjs.com/en/labs/introduction/#usage
